src/modules/Common/components/Datatables/UsernameTable.tsx

```diff
@@ -85,12 +85,32 @@ const UsernameTable = ({
       size: 1,
     },
     {
-      Header: 'Nb of use',
+      Header: (
+        <>
+          Nb of
+          <br />
+          use
+        </>
+      ),
       accessor: 'value',
       align: 'right',
       size: 1,
       Cell: ({ value }: any) => value.toLocaleString('en'),
     },
+    {
+      Header: (
+        <>
+          % of all
```

Comment:
https://github.com/ambanum/information-manipulation-analyzer/pull/62#issuecomment-1021951269

Reply:
I have an error right now so I merge without this change